.github/workflows/nix.yml

```diff
@@ -25,8 +27,8 @@ jobs:
         extra_nix_config: |
           extra-access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
           extra-experimental-features = nix-command flakes ca-derivations
-          extra-substituters = https://cache.nixos.org/
-          extra-trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+          extra-substituters = https://cache.nixos.org/ https://cache.garnix.io/
+          extra-trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g=
           require-sigs = true
           sandbox = true
           extra-system-features = nixos-test benchmark big-parallel kvm
```

Comment:
Seems like none of these are necessary?

Reply:
None of what?

Reply:
> None of what?

Oh sorry, the features. I'm not sure. Try without?

Reply:
I think Gabby will "just know" so I might wait for her input. It is annoying to test these because you have to merge them into main. It is also reasonable to just not worry about it.